src/main/java/com/btk5h/skriptmirror/skript/Types.java

```diff
@@ -214,6 +215,31 @@ public String getVariableNamePattern() {
     Classes.registerClass(new ClassInfo<>(Section.class, "section")
       .user("sections?")
     );
+
+    Classes.registerClass(new ClassInfo<>(Expression.class, "expression")
```

Comment:
skript-reflect actually already returns unwrapped Expression objects, but there hasn't been a registered classinfo for it